sim/dex-data.ts

```diff
@@ -859,6 +854,73 @@ export class Move extends BasicEffect implements Readonly<BasicEffect & MoveData
 	}
 }
 
+export class ActiveMove extends Move {
```

Comment:
Hm I don't think I like the idea of `ActiveMove` being its own class. Especially since it doesn't actually extend `Move`, it actually makes move properties non-readonly.